sys/net/application_layer/rdcli_simple/rdcli_simple_standalone.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_rdcli_simple
+ * @{
+ *
+ * @file
+ * @brief       Standalone extension for the simple RD registration client
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "log.h"
+#include "thread.h"
+#include "xtimer.h"
+#include "net/rdcli_config.h"
+#include "net/rdcli_simple.h"
+
+#define STACKSIZE           (THREAD_STACKSIZE_DEFAULT)
+#define PRIO                (THREAD_PRIORITY_MAIN - 1)
+#define TNAME               "rdcli_simple"
+
+char stack[STACKSIZE];
+
+static void *reg_runner(void *arg)
+{
+    (void)arg;
+
+    /* wait some seconds to give the address configuration some time to settle */
+    xtimer_sleep(3);
```

Comment:
Isn't more useful to have a define for this delay? Are there situations where 3 seconds is insufficient?